pkgs/tools/networking/dd-agent/6.nix

```
@@ -51,6 +51,8 @@ in buildGoPackage rec {
     sed -e "s|PyChecksPath =.*|PyChecksPath = \"$bin/${python.sitePackages}\"|" \
         -e "s|distPath =.*|distPath = \"$bin/share/datadog-agent\"|" \
         -i cmd/agent/common/common_nix.go
+    sed -e "s|/bin/hostname|/run/current-system/sw/bin/hostname|" \
```

```suggestion
    sed -e "s|/bin/hostname|${lib.getBin hostname}/bin/hostname|" \
```

Add both `lib` and `hostname` as arguments.

---

Thanks, fixed.